src/config/dynamic-theme-fixes.config

```diff
@@ -1141,6 +1141,16 @@ IGNORE INLINE STYLE
 
 ================================
 
+fanatical.com
+
+INVERT
+.trustpilot > .logo-container > a > img
+
+IGNORE INLINE STYLE
+*
```

Comment:
Remove this and add `.drm-container-steam` into the INVERT section that should target. That should target the steam logo

Reply:
But then I'll have to do that for other logos such as rockstar games, bethesda and a tonne of others which may come on the front page. fanatical.com has most elements in dark mode by default. So `*` doesn't have adverse effects.

Reply:
I will see if they is a global solution

Reply:
Let's use `.drm-container` I see other element's that need Inline styling.

Reply:
That fixes it.